src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/ResolvedFrameworkReference.xaml

```diff
@@ -8,7 +8,7 @@
   
   <Rule.DataSource>
     <DataSource HasConfigurationCondition="False"
-                ItemType="_ResolvedFrameworkReference"
+                ItemType="FrameworkReference"
```

Comment:
This should probably have been included in #5201. I'm not sure why this property is needed, as it doesn't effect anything as far as I can tell. However other such property-level `ItemType` properties match the outer item type, so I've updated this too for consistency.